addOns/pscanrules/src/test/java/org/zaproxy/zap/extension/pscanrules/InfoPrivateAddressDisclosureScanRuleUnitTest.java

```diff
@@ -160,7 +167,7 @@ void passesIfNonPrivateOrInvalidIp() throws HttpMalformedHeaderException {
             HttpMessage msg = createHttpMessage(candidate);
             scanHttpResponseReceive(msg);
         }
-        assertThat(alertsRaised.size(), equalTo(0));
```

Comment:
Shouldn’t this assertion still be valid?

Reply:
This should be parameterized instead.

Reply:
I've removed it because adds nothing and it's well covered by the following line: